docs/csaf.md

```diff
@@ -0,0 +1,242 @@
+[here](security.access.redhat.com/data).
+
+## Document Structure
+The CSAF standard requires three main sections: document metadata, a product tree and vulnerability metadata.
+The full document structure, without values, can be found
+[here](https://github.com/RedHatProductSecurity/security-data-guidelines/blob/csaf-vex-guidelines/docs/csaf.json).
+
+The following sections break down the information included in VEX documents using the
+[RHSA]() as an example.
+
+### Document Metadata 
+The `document` section contains general information about the published document itself including CVE severity, vendor,
+published date and revision history.
+
+The `aggregate_severity.text` object displays the general CVE severity:
+
+```
+"aggregate_severity": {
+    "namespace": "https://access.redhat.com/security/updates/classification/",
+    "text": "moderate"
+},
+```
+
+The following objects provide general information about the VEX file itself:
+
+```
+"category": "csaf_vex",
+"csaf_version": "2.0",
+"distribution": {
+    "text": "Copyright © Red Hat, Inc. All rights reserved.",
+    "tlp": {
+        "label": "WHITE",
+        "url": "https://www.first.org/tlp/"
+    }
+},
+"lang": "en",
+"notes": [
+    {
+        "category": "legal_disclaimer",
+        "text": "This content is licensed under the Creative Commons Attribution 4.0 International License (https://creativecommons.org/licenses/by/4.0/). If you distribute this content, or a modified version of it, you must provide attribution to Red Hat Inc. and provide a link to the original.",
+        "title": "Terms of Use"
+    }
+],
+```
+
+Vendor information is represented in the `publisher` object:
+```
+"publisher": {
+    "category": "vendor",
+    "contact_details": "https://access.redhat.com/security/team/contact/",
+    "issuing_authority": "Red Hat Product Security is responsible for vulnerability handling across all Red Hat offerings.",
+    "name": "Red Hat Product Security",
+    "namespace": "https://www.redhat.com"
+},
+```
+
+CVE ID, CVE publish date and CVE revision history:
+
+```
+```
+
+### Product Tree 
+The `product_tree` section identifies all fixed Red Hat software, represents the nested 
+relationship of component to product and provides CPEs or PURLs depending on the affected layer. There are two main 
+objects in the `product_tree` object: `branches` and `relationships`.
+
+#### Branches
+The parent `branches` object has one child object of the `vendor` category with the name set to "Red Hat". All 
+affected Red Hat products and components will be nested in that `branches` array. Compressed down, the parent branches 
+object would look like:
+
+```
+{
+  "branches": [
+    {
+      "branches": []
+      "category": "vendor"
+      "name": "Red Hat:
+    }
+  ]
+}
+```
+
+All nested objects included in the `branches` object of the `vendor` category fall into the following subcategories:
```

Comment:
The formatting of the list below is not quite right, please check what it looks like when rendered by mkdocs, because it squashes all of the items into a single paragraph:

![image](https://github.com/user-attachments/assets/25310bb6-06a6-49ec-8e17-0120d4d767d2)